[2025-05-09T07:07:41.699+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-09T07:07:41.741+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: e_commerce_dw_etl_decorator.transform.transform_dim_sellers scheduled__2025-05-08T00:00:00+00:00 [queued]>
[2025-05-09T07:07:41.751+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: e_commerce_dw_etl_decorator.transform.transform_dim_sellers scheduled__2025-05-08T00:00:00+00:00 [queued]>
[2025-05-09T07:07:41.752+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-05-09T07:07:41.769+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform.transform_dim_sellers> on 2025-05-08 00:00:00+00:00
[2025-05-09T07:07:41.778+0000] {standard_task_runner.py:63} INFO - Started process 271 to run task
[2025-05-09T07:07:41.786+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'e_commerce_dw_etl_decorator', 'transform.transform_dim_sellers', 'scheduled__2025-05-08T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/ecommerce_dw_dag.py', '--cfg-path', '/tmp/tmp4cr2i55w']
[2025-05-09T07:07:41.789+0000] {standard_task_runner.py:91} INFO - Job 169: Subtask transform.transform_dim_sellers
[2025-05-09T07:07:41.857+0000] {task_command.py:426} INFO - Running <TaskInstance: e_commerce_dw_etl_decorator.transform.transform_dim_sellers scheduled__2025-05-08T00:00:00+00:00 [running]> on host 335a73e8cf0f
[2025-05-09T07:07:42.344+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='e_commerce_dw_etl_decorator' AIRFLOW_CTX_TASK_ID='transform.transform_dim_sellers' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T00:00:00+00:00'
[2025-05-09T07:07:42.345+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-09T07:07:42.355+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-05-09T07:07:42.375+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-05-09T07:07:42.537+0000] {logging_mixin.py:188} INFO - Transformed and saved data to dim_sellers
[2025-05-09T07:07:42.539+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-09T07:07:42.539+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-09T07:07:42.571+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=e_commerce_dw_etl_decorator, task_id=transform.transform_dim_sellers, run_id=scheduled__2025-05-08T00:00:00+00:00, execution_date=20250508T000000, start_date=20250509T070741, end_date=20250509T070742
[2025-05-09T07:07:42.658+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-09T07:07:42.684+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
